mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -145,7 +145,16 @@ def playlistadd(context, name, uri):
 
         ``NAME.m3u`` will be created if it does not exist.
     """
-    raise exceptions.MpdNotImplemented  # TODO
+    uri = context.lookup_playlist_uri_from_name(name)
+    playlist = uri is not None and context.core.playlists.lookup(uri).get()
+    if not playlist:
+        playlist = context.core.playlists.create(name).get()
```

Comment:
> Otherwise we have no way of creating the playlist in the right backend.

In this case, MPD client specifies to which playlist should track be added, so returning error if playlist can't handle uri is probably only option.